### ISSUE TITLE
Forms results to e-mail

### ISSUE THREAD

**Issue by tbyrd77** [LiveChurchSolutions/ChurchAppsSupport#159]
When a form is completed I'm not sure where the results go. I thought it would go to my e-mail but I can't find it there.

**Comment by Himali-Malvawala** [comment]
yes, you'll receive emails whenever the form is submitted, but for that to work you must have your email notifications "On" for the forms that you want to receive emails for. 

You can set those on your form's page under the "Form Members" tab.

**Comment by tbyrd77** [comment]
I double checked and I do have the "On" button turned on for email
notifications.  I'm not sure if I'm doing something wrong but I'm still not
getting the completed form in my inbox.. Let me know if you figure out what
I'm doing wrong.  Thanks

**Comment by jzongker** [comment]
@tbyrd77 Is this still not working for you.  It's possible everything hadn't been deployed to production yet when you originally tried three weeks ago.  I just tested now and I received the notification and was able to see the submission in CHUMS as well.

**Comment by tbyrd77** [comment]
Yes it is working well. Thanks. Sent from my iPhoneOn Jun 14, 2023, at 12:57 PM, Jeremy Zongker ***@***.***> wrote:﻿
@tbyrd77 Is this still not working for you.  It's possible everything hadn't been deployed to production yet when you originally tried three weeks ago.  I just tested now and I received the notification and was able to see the submission in CHUMS as well.

**Comment by tbyrd77** [comment]
Sorry I didn’t get back to you. I talked to Micheal and thought he would pass word on to you. Thanks. Sent from my iPhoneOn Jun 14, 2023, at 12:57 PM, Jeremy Zongker ***@***.***> wrote:﻿
@tbyrd77 Is this still not working for you.  It's possible everything hadn't been deployed to production yet when you originally tried three weeks ago.  I just tested now and I received the notification and was able to see the submission in CHUMS as well.
